<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SetProfile</class>
    <widget class="QMainWindow" name="SetProfile">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>286</width>
                <height>177</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="maximumSize">
            <size>
                <width>286</width>
                <height>229</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Profile</string>
        </property>
        <property name="windowIcon">
            <iconset resource="resources.qrc">
                <normaloff>:/resource/profile-large.png</normaloff>:/resource/profile-large.png
            </iconset>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QGridLayout" name="gridLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>Nickname</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>Timezone</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <layout class="QVBoxLayout" name="verticalLayout_2">
                                <item>
                                    <widget class="QLabel" name="label_4">
                                        <property name="text">
                                            <string>Avatar</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>Location</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" name="location"/>
                        </item>
                        <item row="3" column="1">
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QLabel" name="avatarDisplay">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>48</width>
                                                <height>48</height>
                                            </size>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>48</width>
                                                <height>48</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string/>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <spacer name="horizontalSpacer">
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                            <size>
                                                <width>40</width>
                                                <height>20</height>
                                            </size>
                                        </property>
                                    </spacer>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="setAvatar">
                                        <property name="text">
                                            <string>Choose avatar...</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="nickname"/>
                        </item>
                        <item row="2" column="1">
                            <widget class="QComboBox" name="timezone">
                                <item>
                                    <property name="text">
                                        <string>-1200</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-1100</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-1000</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-930</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-900</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-800</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-700</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-600</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-500</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-400</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-330</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-300</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-200</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>-100</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>000</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+100</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+200</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+300</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+330</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+400</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+430</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+500</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+530</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+545</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+600</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+630</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+700</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+800</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+830</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+845</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+900</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+930</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+1000</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+1030</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+1100</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+1200</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+1245</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+1300</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>+1400</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QDialogButtonBox" name="buttonBox">
                        <property name="standardButtons">
                            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <tabstops>
        <tabstop>nickname</tabstop>
        <tabstop>location</tabstop>
        <tabstop>timezone</tabstop>
        <tabstop>setAvatar</tabstop>
    </tabstops>
    <resources>
        <include location="resources.qrc"/>
    </resources>
    <connections/>
</ui>
